java/read-symbols-mobile-style-file/src/main/java/com/esri/arcgisruntime/sample/readsymbolsmobilestylefile/MainActivity.java

```diff
@@ -110,21 +111,13 @@
     mGraphicsOverlay = new GraphicsOverlay();
     mMapView.getGraphicsOverlays().add(mGraphicsOverlay);
 
-    // add listener to handle motion events when the user taps on the map view
-    mMapView.setOnTouchListener(
-        new DefaultMapViewOnTouchListener(this, mMapView) {
-          @Override
-          public boolean onSingleTapConfirmed(MotionEvent motionEvent) {
-            addGraphic(mGraphicsOverlay, mapPointFrom(mMapView, motionEvent), mCurrentMultilayerSymbol);
-            return true;
-          }
-        });
-
     // add a seek bar to change the size of the current multilayer symbol
-    SeekBar sizeSeekBar = findViewById(R.id.sizeSeekBar);
+    mSizeSeekBar = findViewById(R.id.sizeSeekBar);
+    // disable seek bar until read permission is granted
+    mSizeSeekBar.setEnabled(false);
```

Comment:
You could disable the Views in XML opposed to in Java, could reduce LOC count in `MainActivity`.